src/components/Reviews/ReviewVoteButton.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+import { incrementVotes } from "../../../Utils/fetchUtils";
+
+function ReviewVoteButton({ currReview }) {
+    const [votes, SetVotes] = useState(currReview[0].votes);
+
+    //add user specific disables in BE!
+    const [upDisabled, SetUpDisabled] = useState(false)
+    const [downDisabled, SetDownDisabled] = useState(false)
```

Comment:
We also want to have a loading state for our comments. On a very slow network, the article can load first and the comments afterwards.